bokeh/command/__main__.py

```diff
@@ -0,0 +1,8 @@
+'''
+
+'''
+import sys
+
+from .bootstrap import main
+
+main(sys.argv)
```

Comment:
Almar suggested a `bokeh/__main__.py` and said `bokeh/command/__main__.py` wouldn't work, I guess for `python -m`; I don't really have a full understanding of how `python -m` works, but should we move this up one?